tools/names.js

```diff
@@ -15,13 +15,20 @@ const getName = (name, file) => {
         throw new Error(`No name found in ${file}, probably it is not a valid player page`);
     }
 
-    // Handles rare first names with spaces manually, along with Nene having no last name and some typos
+    // Handles rare first names manually:
+    // 1. names with spaces
+    // 2. Nene having no last name
+    // 3. Apostrophe first names
```

Comment:
Why handle that here? Why not a fnFixes thing? The last names don't matter. I added this part specifically for names where the first and last names were not distinguishable.